src/main/java/uk/gov/hmcts/reform/sscs/messaging/HmcHearingsTopicListenerProperties.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscs.messaging;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class HmcHearingsTopicListenerProperties {
+
+    private String connectionString;
+    private String topicName;
+    private String subName;
```

Comment:
These properties will need to be set as env vars and injected via @Value using the application.yaml file

Reply:
Yes we can also have them as env variable and capture them with @value

Reply:
There is a need to do it now